src/Components/Facility/DuplicatePatientDialog.tsx

```diff
@@ -27,99 +18,114 @@ const DuplicatePatientDialog = (props: Props) => {
   const [action, setAction] = useState("");
 
   return (
-    <Dialog open={true} maxWidth={"sm"}>
-      <DialogTitle
-        className=" font-semibold text-3xl"
-        id="font-semibold text-3xl"
-      >
-        Patient Records Found
-      </DialogTitle>
-      <DialogContent>
-        <div className="grid gap-4 grid-cols-1">
-          <div>
-            <p className="leading-relaxed text-sm">
-              It appears that there are patient records that contain the same
-              phone number as the one you just entered. (
-              <span className="font-bold">{patientList[0].phone_number}</span>)
-            </p>
+    <DialogModal
+      title={<FieldLabel className="text-lg">Patient Records Found</FieldLabel>}
```

Comment:
@Ashesh3 the DialogModal already applies styling for the title. Was the title styling changed intentionally?
```suggestion
      title="Patient Records Found"
```

Reply:
@rithviknishad I didn't double check it, just took the template from `FacilitiesSelectDialogue.tsx` https://github.com/coronasafe/care_fe/blob/develop/src/Components/ExternalResult/FacilitiesSelectDialogue.tsx#L21
I'll go ahead and remove it here.